File: online-judge-solutions/Codeforces/448A.py
#codeforces.com/problemset/problem/448/A
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
import math
a=sum(map(int,input().split()))
b=sum(map(int,input().split()))
n=int(input())
print("NO" if (math.ceil(a/5)+math.ceil(b/10)>n) else "YES")

File: online-judge-solutions/Codeforces/448A.cpp
//codeforces.com/problemset/problem/448/A/
//Lang		:	GNU C++14
//Time		:	15 ms
//Memory	:	100 KB
#include<iostream>
#include<math.h>
using namespace std;
int main(){
	double a1,a2,a3,b1,b2,b3,n;
	while(cin>>a1>>a2>>a3>>b1>>b2>>b3>>n){
		cout<<(((ceil((a1+a2+a3)/5)+ceil((b1+b2+b3)/10))>n)?"NO":"YES")<<endl;
	}
	return 0;
}
